src/repror/internals/db.py

```diff
@@ -7,7 +7,7 @@
 import os
 from pathlib import Path
 import tempfile
-from typing import Generator, Optional
+from typing import Generator, Literal as Lit, Optional
```

Comment:
why Literal as Lit?

Reply:
Because there is another Literal import beneath it.